src/audio/google_rtc_audio_processing.c

```diff
@@ -371,50 +371,59 @@ static int google_rtc_audio_processing_copy(struct comp_dev *dev)
 
 	buffer_stream_invalidate(cd->aec_reference, num_aec_reference_bytes);
 
-	aec_reference_buff_frag = 0;
-	for (frame = 0; frame < num_aec_reference_frames; frame++) {
-		for (channel = 0; channel < cd->num_aec_reference_channels; ++channel) {
-			src = audio_stream_read_frag_s16(&cd->aec_reference->stream,
-							 aec_reference_buff_frag + channel);
-			cd->aec_reference_buffer[cd->num_aec_reference_channels *
-				cd->aec_reference_frame_index + channel] = *src;
+	remain = num_aec_reference_frames * cd->num_aec_reference_channels;
```

Comment:
Isn't this just samples?

Reply:
Yes, it is I was wondering how to name this. In first loop it's samples, in second loop it's frames. It seems that the optimizer makes variables more local so it should be the same to have two variables without increase in stack usage: samples and frames.